demo/top_down_pose_tracking_demo_with_mmdet.py

```diff
@@ -63,7 +63,7 @@ def main():
         '--bbox-thr',
         type=float,
         default=0.3,
-        help='Bounding box score threshold')
+        help='Bounding box score threshold or OKS score threshlod')
```

Comment:
It would be clearer if we add another arg for `oks-thr`.

Reply:
OK. I'll fix it right away.

Reply:
I think I annotated the wrong place.

Reply:
I'll set the top down use IOU and bottom up use OKS as the default.